poolmgr/gp.go

```diff
@@ -478,7 +487,7 @@ func (gp *GenericPool) GetFuncSvc(m *api.ObjectMeta) (*funcSvc, error) {
 	if gp.useSvc {
 		svcName := fmt.Sprintf("svc-%v", m.Name)
 		if len(m.UID) > 0 {
-			svcName += ("-" + string(m.UID))
+			svcName = fmt.Sprintf("%s-%v", svcName, m.UID)
```

Comment:
curious about this change -- are you actually using it? I've been meaning to remove this code, it's dead code anyway.